### PR TITLE
Add Survey MBS 0106

### PR DESCRIPTION
**Pull request by nealedj**
### What is the context of this PR?
New survey on for MBS 0106 added to SR

### How to review 
Check content on slides match: 
https://trello.com/c/VGKHp3ZQ/2026-turnover-mbs-form-0106-m

### CODE REVIEW

File: data/en/mbs_0106.json [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Turnover",
```
Comment (shahi645) [inline data/en/mbs_0106.json]: Should be `Total turnover`

File: data/en/mbs_0106.json [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+                                    "content": [{
+                                            "description": "Include:",
+                                            "list": [
+                                                "exports",
+                                                "payment for work in progress",
```
Comment (shahi645) [inline data/en/mbs_0106.json]: `payments`

File: data/en/mbs_0106.json [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+                    {
+                        "id": "turnover-block",
+                        "type": "Question",
+                        "questions": [{
+                            "guidance": {
+                                "content": [{
+                                        "title": "Include:",
+                                        "list": [
+                                            "exports",
+                                            "payment for work in progress",
```
Comment (shahi645) [inline data/en/mbs_0106.json]: payments

**Review by shahi645 — CHANGES_REQUESTED** [review]
only a couple of minor content changes

**Review by shahi645 — APPROVED** [review]

File: data/en/mbs_0106.json [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+                                                "income from the sales of fixed capital assets",
+                                                "grants and subsidies",
+                                                "insurance claims",
+                                                "interest received"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                {
+                                    "question": "Changes in turnover",
```
Comment (dwyeradam) [inline data/en/mbs_0106.json]: Changes in `total` turnover
Slides incorrect.

File: data/en/mbs_0106.json [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+                                            "insurance claims",
+                                            "interest received"
+                                        ]
+                                    }
+                                ]
+                            },
+                            "id": "turnover-question",
+                            "answers": [{
+                                "id": "turnover-answer",
+                                "label": "Turnover excluding VAT",
```
Comment (dwyeradam) [inline data/en/mbs_0106.json]: `Total` turnover excluding VAT

**Review by dwyeradam — CHANGES_REQUESTED** [review]
Just little niggles.

**Review by dwyeradam — APPROVED** [review]

**Comment by dwyeradam** [comment]
@MIslam92 Squishy squashy.

**Pull request merged**